src/physics/surface_fluxes/heat.jl

```diff
@@ -49,10 +49,10 @@ SurfaceOceanHeatFlux(SG::SpectralGrid; kwargs...) = SurfaceOceanHeatFlux{SG.NF}(
 initialize!(::SurfaceOceanHeatFlux, ::PrimitiveEquation) = nothing
 
 function surface_heat_flux!(
-    column::ColumnVariables,
+    column::ColumnVariables{NF},
```

Comment:
I'm wondering whether he should generally use `eltype(column)` instead of the `where NF` syntax?

Reply:
Currently we don't have `eltype(column)` explicitly defined, so it yields `Any`